components/card/card.css

```diff
@@ -0,0 +1,138 @@
+.card{
+    display: flex;
+    border: 1px solid var(--primary-color);
+}
+.card-vertical{
+    flex-direction: column;
+    position: relative;
+    max-width: 302px;
+}
+.card-horizontal{
+    flex-direction: row;
+    max-width: 100%;
+}
+.card-vertical-image{
+    position: relative;
+}
+.card-horizontal-image{
+   max-width: 200px;
+}
+.card-img{
+    max-width: 100%;
+    height: 100%;
+}
+.wishlist-icon{
+    position: absolute;
+    top: 5px;
+    right: 5px;
+    padding: 5px;
+    border-radius: 50%;
+    color: var(--text-color-2);
+    background-color: var(--bg-color-1);
+    cursor: pointer;
+}
+.wishlist-icon:hover{
+    color: var(--primary-color);
+}
+
+.card-content{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.card-content-horizontal{
+    margin: 5px 20px;
+}
+.card-heading{
+    margin: 10px 5px;
+}
+.card-heading h3{
+    font-size: 1.25rem;
+    font-weight: 500;
+    margin-bottom: 10px;
+    letter-spacing: 2px;
+}
+.card-heading p{
+    font-size: 14px;
+    letter-spacing: 1px;
+}
+
+.card-product-price{
+    display: flex;
+    margin: 10px 5px;
+}
+.product-discount-price{
+    font-weight: 600;
+    margin-right: 10px;
+}
+.product-original-price{
+    text-decoration: line-through;
+}
+
+.product-discount-percent{
+    font-size: 1rem;
+    color: var(--success-dark-color);
+    margin:0px 5px 10px 5px;
+    font-weight: 500;
+}
+.card-btn{
+    padding: 10px 0px;
+    background-color: var(--primary-color);
+    border: 1px solid var(--primary-color);
+    cursor: pointer;
+    width: 100%;
+    color: var(--text-color-1);
+    font-weight: 500;
+    font-size: 20px;
+}
+.card-btn:hover{
+    background-color: transparent;
+    color: var(--primary-color);
+    border: 1px solid var(--primary-color);
+}
+.card-btn-icon{
+    margin-right:12px;
+}
+.card-secondary-btn{
+    margin-top: 10px;
+    padding: 10px;
+    background-color: var(--secondary-color);
+    border: 1px solid var(--secondary-color);
+    cursor: pointer;
+    width: 100%;
+    color: var(--text-color-1);
+    font-weight: 500;
+    font-size: 20px;
+}
+.card-secondary-btn:hover{
+    opacity: 0.6;
+}
+
+.card-image-badge{
+    position: absolute;
+    top:0px;
+    left: 0px;
+    padding: 5px;
+    font-weight: 500;
+    letter-spacing: 2px;
+    background-color: var(--primary-color);
+    color: var(--text-color-1);
+}
+.card-text-overlay{
+    display: flex;
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    z-index: 1;
+    background-color: rgba(0, 0, 0, 0.8);
+}
+.card-text-overlay>span{
+    color: var(--text-color-1) ;
+    font-size: 3rem;
+    margin: auto;
+    background-color: rgba(2, 129, 149, 0.5);
+}
+
+.card-shadow{
+    box-shadow: 10px 10px 50px rgb(0, 0, 0, 0.8);
```

Comment:
Try to use color variables instead of hard-coding them.

Reply:
changes done